com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -96,19 +96,21 @@ The **display stats** panel is only visible in play mode and can be used to debu
+| **GBuffer**                                      | Use the drop-down to select a property to visualize from the GBuffer for deferred Materials. |
+| **Material Validator**                           | Use the drop-down to select properties to show validation colors for.<br/>&#8226; **Diffuse Color**: Select this option to check if the diffuse colors in your Scene adheres to an acceptable [PBR](Glossary.md#PhysicallyBasedRendering) range. If the Material color is out of this range, the debugger displays it in the **Too High Color** color if it is above the range, or in the **Too Low Color** if it is below the range.<br/>&#8226; **Metal or SpecularColor**: Select this option to check if a pixel contains a metallic or specular color that adheres to an acceptable PBR range. If it does not, the debugger highlights it in the **Not A Pure Metal Color**.For information about the acceptable PBR ranges in Unity, see the [Material Charts documentation](https://docs.unity3d.com/Manual/StandardShaderMaterialCharts.html). |
+| **- Too High Color**                             | Use the color picker to select the color that the debugger displays when a Material's diffuse color is above the acceptable PBR range.<br/>This property only appears when you select **Diffuse Color** or **Metal or SpecularColor** from the **Material Validator** drop-down. |
+| **- Too Low Color**                              | Use the color picker to select the color that the debugger displays when a Material's diffuse color is below the acceptable PBR range.<br/>This property only appears when you select **Diffuse Color** or **Metal or SpecularColor** from the **Material Validator** drop-down. |
+| **- Not A Pure Metal Color**                     | Use the color picker to select the color that the debugger displays if a pixel defined as metallic has a non-zero albedo value. The debugger only highlights these pixels if you enable the **True Metals** checkbox.<br/>This property only appears when you select **Diffuse Color** or **Metal or SpecularColor** from the **Material Validator** drop-down. |
+| **- Pure Metals**                                | Enable the checkbox to make the debugger highlight any pixels which Unity defines as metallic, but which have a non-zero albedo value. The debugger uses the **Not A Pure Metal Color** to highlight these pixels.<br/>This property only appears when you select **Diffuse Color** or **Metal or SpecularColor** from the **Material Validator** drop-down. |
+| **Override Global Material Texutre Mip Bias**    | *NOTE: feature still under development, for now it just works on deferred rendering passes* Enable the checkbox to override the mip bias of texture samplers in material shaders. Use the 'Debug Global Material Texture Mip Bias Value' to control the mip bias override. |
```

Comment:
>for now it just works on deferred rendering passes*
what do you mean? it is expected to work in forward as well. You can't add "NOTE" like this. 
"NOTE: feature still under development," is usually not something we write in doc AFAIK (cc @JordanL8 ). 

However we should mention that it is not compatible with virtual texturing

Reply:
Yeah, we don't do it all in caps https://unity-docs.gitbook.io/style-guide/standard-notes-and-information/notes-and-info-boxes. Also we document the current state of the feature and tend not to mention things like "feature still under development". I don't remember seeing this entry originally so I'll have a look and make suggestions now

Reply:
Enable the checkbox to override the mipmap level bias of texture samplers in material shaders. Use the **Debug Global Material Texture Mip Bias Value** to control the mipmap level bias override.<br/>When using this feature, be aware of the following:<br/>&#8226; It only works in deferred mode.<br/>&#8226; It does not support virtual texturing.<br/>&#8226; It does not affect custom passes.

Markdown:
`Enable the checkbox to override the mipmap level bias of texture samplers in material shaders. Use the **Debug Global Material Texture Mip Bias Value** to control the mipmap level bias override.<br/>When using this feature, be aware of the following:<br/>&#8226; It only works in deferred mode.<br/>&#8226; It does not support virtual texturing.<br/>&#8226; It does not affect custom passes.`

Reply:
I put only works in deferred mode as a placeholder, can you elaborate on the 
> affect the Material pass (i.e Decal, GBuffer and forward).
bit?

Reply:
No more need for NOTE, as I have supported everything that needs support. @sebastienlagarde check the Apply/Reset functions I added surrounding the interesting passes. 

The final proposal for this doc line is then this one:
| **Override Global Material Texutre Mip Bias**    | Enable the checkbox to override the mipmap level bias of texture samplers in material shaders. Use the **Debug Global Material Texture Mip Bias Value** to control the mipmap level bias override.<br/>When using this feature, be aware of the following:<br/>&#8226; It only affects gbuffer, forward opaque, transparency and decal passes. <br/>&#8226; It does not affect virtual texturing sampling.<br/>&#8226; It does not affect custom passes. |

Let me know if this all works.